### PR TITLE
Analytics UI

### PR DESCRIPTION
**Pull request by lfryc**
https://issues.jboss.org/browse/AGPUSH-1384

@edewit could you please review / test this?

### CODE REVIEW

**Comment by matzew** [comment]
the dark blue says 'Push Notifications', which is IMO not correct.

It's the number of targeted devices, for the push

File: jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/metrics/PushMetricsEndpoint.java [hunk 1]
```diff
@@ -85,11 +89,14 @@ public Response pushMessageInformationPerVariant(
             return Response.status(Response.Status.NOT_FOUND).entity("Could not find requested information").build();
         }
 
-        PageResult<PushMessageInformation> pageResult =
+        PageResult<PushMessageInformation, MessageMetrics> pageResult =
                 metricsService.findAllForVariant(id, extractSortingQueryParamValue(sorting), page, pageSize);
```
Comment (matzew) [inline jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/metrics/PushMetricsEndpoint.java]: This code, that is around the "findAllForVariant" is not executed w/ the new UI.

See: https://github.com/matzew/aerogear-unifiedpush-server/commit/3d3e682740a20d6c11a00db0b20ec8bca244a2bb
and: https://github.com/matzew/aerogear-unifiedpush-server/commit/0a8e3b56307dce5680cf209da5164c1e9d76948c

I am fine w/ doing a clean up AFTER this.   Or, @lfryc feel free to do on this PR as well

**Comment by edewit** [comment]
:eyeglasses:

**Comment by edewit** [comment]
the ? icons don't work other then that looks good :+1:

**Comment by matzew** [comment]
thanks, @lfryc landed in master w/ discussed changes

**Pull request merged**